content/brazilian-portuguese/linux-basics/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Minicurso básico de Linux"
+description: "Um guia introdutório sobre os comandos básicos do Linux."
+date: 2022-09-21
+prereq: "Nenhum."
+difficulty: "Intermediário"
+download: ""
+draft: false
+icon: "fas fa-microchip"
+weight: 1
+---
+
+{{% notice warning %}}
+Antes de prosseguirmos, vamos aos requisitos necessários para completar nosso curso com sucesso:
+
+- Um computador que **não** seja do tipo Chromebook
+- Seu computador precisa ter pelo menos 3GB de memória livres
+- Uma boa conexão com a internet para baixar os programas necessários:
+  - ~10 minutos para baixar VirtualBox
+  - ~30 minutos para baixar a imagem de uma máquina virtual
+
+{{% /notice %}}
+
+<iframe style="display: block; margin: auto;" width="560" height="315" src="https://www.youtube.com/embed/zfdlPZYlgtk" alt="A YouTube video that introduces the Linux Workshop" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
I noticed you translated a few other "alt" attributes, so you may want to translate this too:
```suggestion
<iframe style="display: block; margin: auto;" width="560" height="315" src="https://www.youtube.com/embed/zfdlPZYlgtk" alt="Um video do YouTube que apresenta o Minicurso de Linux" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```